src/pages/organizations/[emailAccessToken].tsx

```diff
@@ -0,0 +1,30 @@
+import { NextPage } from "next";
+import { observer } from "mobx-react";
+import { useEffect } from "react";
+import { useRouter } from "next/router";
+import { useStores } from "@/stores/context";
+
+const EmailAccessToken: NextPage = observer(() => {
+  const { organizationStore } = useStores();
+  const router = useRouter();
+  const AccessToken = router.query.emailAccessToken;
+
+  useEffect(() => {
+    if (typeof AccessToken === "string") {
+      organizationStore.verifyEmailConfirm(AccessToken);
+    }
+  }, [AccessToken]);
+
+  return (
+    <>
+      {organizationStore.successMessage === "" ? null : (
+        <h1>{organizationStore.successMessage}</h1>
+      )}
+      {organizationStore.errorMessage === "" ? null : (
+        <h1>{organizationStore.errorMessage}</h1>
+      )}
```

Comment:
해당 메시지들이 없을 때는 undefined로 두는 것이 더욱 명확합니다. 이렇게 하기 위해서는 store에 필드 선언을 `string | undefined` 타입으로 수정해야할 것같네요. 
```suggestion
      {organizationStore.successMessage === undefined ? null : (
        <h1>{organizationStore.successMessage}</h1>
      )}
      {organizationStore.errorMessage === undefined ? null : (
        <h1>{organizationStore.errorMessage}</h1>
      )}
```